ShippingApp/src/main/java/com/javaschool/controller/ShippingController.java

```diff
@@ -0,0 +1,37 @@
+package com.javaschool.controller;
+
+import com.javaschool.service.ShippingService;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.List;
+
+@RestController
+public class ShippingController {
+
+    private ShippingService shippingService;
+
+    public ShippingController(final ShippingService shippingService) {
+        this.shippingService = shippingService;
+    }
+
+    @RequestMapping("/size")
+    public List<String> getSize() {
```

Comment:
What is the difference between using `ResponseEntity`, which you are using on `CustomExceptionHandler.java` and just returning a `List` of objects?

Reply:
The difference is that when returning a list of objects, the error is displayed in the UI. Using ResponseEnity does not display the error and an option appears on the list. Example: [object Object]. What is the expected behavior?